pack/ur.json

```diff
@@ -53,6 +53,24 @@
         "type_code": "hardware",
         "uniqueness": true
     },
+    {
+        "code": "26069",
+        "cost": 0,
+        "deck_limit": 3,
+        "faction_code": "anarch",
+        "faction_cost": 1,
+        "flavor": "Win a new friend today!",
+        "illustrator": "Elizaveta Sokolova",
+        "keywords": "Chip",
+        "pack_code": "ur",
+        "position": 69,
+        "quantity": 3,
+        "side_code": "runner",
+        "text": "[trash]: Set aside the top 6 cards of your stack. You may install 1 program or <strong>virtual</stron> resource from among the set aside cards, paying 2{credit] less. Shuffle 3 of the remaining cards into your stack, then remove the rest from the game.",
```

Comment:
```suggestion
        "text": "[trash]: Set aside the top 6 cards of your stack. You may install 1 program or <strong>virtual</strong> resource from among the set aside cards, paying 2[credit] less. Shuffle 3 of the remaining cards into your stack, then remove the rest from the game.",
```

Reply:
Seems I can't apply my own suggestion - over to you @NoahTheDuke or @gereons I guess?

Reply:
Doesn't work for me either - I guess it's because none of us have write permission on Noah's fork. If we want to get this on NRDB before Noah wakes up, it's probably easiest to create a new PR from your fork.

Reply:
S'all good, we can wait for @NoahTheDuke, no worries.

Reply:
I’m awake! I’ll get it up once I hit my desk in an hour.